posthog/api/dashboard.py

```diff
@@ -61,6 +62,13 @@ class Meta:
         ]
         read_only_fields = ["creation_mode", "effective_restriction_level", "is_shared"]
 
+    def validate_description(self, value: str) -> str:
+        team: Team = Team.objects.get(id=self.context["team_id"])
+        if value and AvailableFeature.DASHBOARD_COLLABORATION not in team.organization.available_features:
+            raise PermissionDenied("You must have paid for dashboard collaboration to set the dashboard description")
+
+        return value
```

Comment:
Will this add N+1 team queries if requesting e.g. a list of dashboards? We do query `https://app.posthog.com/api/projects/$our_team/dashboards/?limit=300` whenever you open the app.

Reply:
Interestingly there must be magic and it's already nplus1. Adding a query to pre-load the team doesn't reduce the number of queries. In. fact there's one _more_ query if I pre-query for the team 🤷‍♂️

Added a test to snapshot and it's the sharing configuration that adds 1 query per dashboard.

Reply:
Django is weird. Found and fixed the source of the nplus1...